addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1278,16 +1275,13 @@ const Wysiwyg = Widget.extend({
         // selection when the modal is closed.
         const restoreSelection = preserveCursor(this.odooEditor.document);
 
-        const $editable = $(OdooEditorLib.closestElement(range.startContainer, '.o_editable') || this.odooEditor.editable);
-        const model = $editable.data('oe-model');
-        const field = $editable.data('oe-field');
-        const type = $editable.data('oe-type');
+        const editable = OdooEditorLib.closestElement(range.startContainer, '.o_editable') || this.odooEditor.editable;
+        const {resModel, resId, field, type } = this._getRecordInfo(editable);
```

Comment:
Wait, this is not equivalent. jQuery `.data` is not the same thing as the DOM `dataset`.

Reply:
I know that! But I thought those were part of the element's dataset (data-* attributes), so we woudn't need to fetch them using jQuery. Like in:
![image](https://github.com/odoo/odoo/assets/65516503/583f1208-f817-424d-a45b-660e2ac7ccd0)

It seemed OK in my tests, and I though that reducing dependence on jQuery was always welcome.
BUT now I'm not completely sure this won't break anything, somewhere...
I'm taking the conservative approach and reverting it to jQuery `.data` instead. (done/pushed)